### DISCUSSION TITLE
Kvantum and krusader transparency

### DISCUSSION THREAD

**Discussion by adremides** [tsujan/Kvantum#989] [Q&A]
I been able to configure kvantum to make most part of the app transparent. But some of the widgets seems to not be affected by kvantum theme.
There is some way to make the title panel and panel itself (list of files and status) transparent too?
![Screenshot_krusader](https://github.com/tsujan/Kvantum/assets/19670432/d961aa9a-2cea-4304-a1bb-b42a6dec2b1f)

**Comment by tsujan** [comment]
I'm afraid not every widget can be made transparent. Moreover, some apps may override the Qt style by filling some widgets with colors in their codes.

**Reply by adremides** [reply]
Hi! Thanks for the quick response. I made a [post in KDE Forum](https://discuss.kde.org/t/krusader-and-kvantum-transparency/17533) where someone suggested altering the color scheme to add an alpha channel, and it worked.

I'm not sure if it's worth mentioning here because I don't fully understand how Kvantum handles those settings (if it can).

**Reply by tsujan** [reply]
> someone suggested altering the color scheme to add an alpha channel, and it worked.

It shows that the app uses the KDE color scheme directly in its code, instead of leaving the colors to the Qt widget style. IMO, that's a bad design, but several KDE apps do it.
